easybuild/easyconfigs/p/PPanGGOLiN/PPanGGOLiN-1.1.136-foss-2020b.eb

```diff
@@ -0,0 +1,74 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Denis Kristak
+easyblock = 'PythonPackage'
+
+name = 'PPanGGOLiN'
+version = '1.1.136'
+
+homepage = 'https://github.com/labgem/PPanGGOLiN'
+description = """
+PPanGGOLiN is a software suite used to create and manipulate prokaryotic
+pangenomes from a set of either genomic DNA sequences or provided genome annotations. It is designed
+to scale up to tens of thousands of genomes. It has the specificity to partition the pangenome using
+a statistical approach rather than using fixed thresholds which gives it the ability to work with low-quality
+data such as Metagenomic Assembled Genomes (MAGs) or Single-cell Amplified Genomes (SAGs) thus taking advantage
+of large scale environmental studies and letting users study the pangenome of uncultivable species.
+"""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+source_urls = ['https://github.com/labgem/%(name)s/archive/refs/tags/']
+sources = ['%(version)s.tar.gz']
+patches = ['PPanGGOLiN-1.1.136_new_compiler_compatibility.patch']
```

Comment:
please use `%(version)s` rather than hardcoding the version in the patch filename

Reply:
done